controller/auth/user.go

```diff
@@ -0,0 +1,54 @@
+package auth
+
+import (
+	"strconv"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"golang.org/x/crypto/bcrypt"
+
+	"github.com/gorilla/mux"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/HackGT/SponsorshipPortal/model/user"
+)
+
+type userController struct {
+	db *sqlx.DB
+}
+
+type User struct {
+	Email    string
+	Password string
+	Org_id   int64
+}
+
+func (u userController) Create(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	var jsonUser User
+	response, _ := ioutil.ReadAll(r.Body)
+	err := json.Unmarshal(response, &jsonUser)
+	if err != nil {
+		w.Write([]byte(string(response) + "\nMalformed JSON Request"))
```

Comment:
Also, import `log "github.com/sirpusen/logrus"` and then do `log.WithError(err).Warn("Message here")`. It uses this library: https://github.com/sirupsen/logrus